im_user.profile.inc

```diff
@@ -34,12 +28,12 @@ function _get_profile_name($profile) {
  * @param XML response of WS:getProfil 
  */
 function _is_carrefour_market($profile) {  
-  if(!isset($profile->grp)) return FALSE;
-  foreach ($profile->grp as $grp){
+  if (!isset($profile->grp)) return FALSE;
+  foreach ($profile->grp as $grp) {
     $counter = 0;
     $level   = '';
     $cn      = ''; 
-    foreach($grp->attr as $attr){
+    foreach ($grp->attr as $attr) {
   	  if((string)$grp->attr[$counter]['type'] == 'level' && (string)$grp->attr[$counter] == 'GRPOPE') {

```

Comment:
this looks like a weird indentation - I suspect you haven't configured your IDE to automatically re-format your code and you have done it all by hand... is this the case?